WordPress/Classes/Extensions/UIView+Autolayout.swift

```diff
@@ -0,0 +1,14 @@
+import UIKit
+
+extension UIView {
+
+    @objc func pinSubviewToAllEdges(_ subview: UIView,
```

Comment:
Maybe this is still on your TODO list, but this method doesn't appear to be used.

Reply:
oh yes I had it called from somewhere but then I didn't have to use it anymore, I just didn't remove it since it is an handy function, I can quickly remove it if you think it is not that useful.

Reply:
Yea, for the sake of maintenance and clarity, if it's not used it'd be better if it not hang around.

Reply:
Removed